### ISSUE TITLE
Doesn't work with typescript - cannot find nama 'array'

### ISSUE THREAD

**Issue by ilyaLibin** [ealush/vest#136]
![image](https://user-images.githubusercontent.com/12880486/81266016-b661d300-904c-11ea-9d6c-147993c487a8.png)

"typescript": "~3.8.3",

**Comment by ilyaLibin** [comment]
```

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:20:30 - error TS2304: Cannot find name 'array'.

20   longerThan(value: string | array): EnforceRules;
                                ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:21:38 - error TS2304: Cannot find name 'array'.

21   longerThanOrEquals(value: string | array): EnforceRules;
                                        ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:22:32 - error TS2304: Cannot find name 'array'.

22   numberEquals(value: string | array): EnforceRules;
                                  ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:23:35 - error TS2304: Cannot find name 'array'.

23   numberNotEquals(value: string | array): EnforceRules;
                                     ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:24:31 - error TS2304: Cannot find name 'array'.

24   shorterThan(value: string | array): EnforceRules;
                                 ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:25:39 - error TS2304: Cannot find name 'array'.

25   shorterThanOrEquals(value: string | array): EnforceRules;
                                         ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:28:26 - error TS2304: Cannot find name 'array'.

28   inside(value: string | array): EnforceRules;
                            ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:29:29 - error TS2304: Cannot find name 'array'.

29   notInside(value: string | array): EnforceRules;
                               ~~~~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:63:30 - error TS1016: A required parameter cannot follow an optional parameter.

63   done: (fieldName?: string, cb: (output: VestOutput) => void) => VestOutput;
                                ~~

../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts:81:5 - error TS1016: A required parameter cannot follow an optional parameter.

81     cb: Function
       ~~


Found 10 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! accounting@1.0.0 build: `rm -rf ./dist && ./node_modules/typescript/bin/tsc`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the accounting@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

**Comment by ilyaLibin** [comment]
>>> accounting
Found mismatch: "services/accounting/controllers/JournalEntriesController.js": "9448056c0b400a8b443257ec18f17bb1aca6a356" !== "undefined"
rm -rf ./dist && ./node_modules/typescript/bin/tsc
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(20,30): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(21,38): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(22,32): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(23,35): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(24,31): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(25,39): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(28,26): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(29,29): error TS2304: Cannot find name 'array'.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(63,30): error TS1016: A required parameter cannot follow an optional parameter.
../../common/temp/node_modules/.registry.npmjs.org/vest/1.0.9/node_modules/vest/vest.d.ts(81,5): error TS1016: A required parameter cannot follow an optional parameter.

**Comment by ealush** [comment]
Fixed in 1.10.
